automated/linux/pibridge/test-pb-1.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
```

Comment:
the link to the picture of connections of DUT can be added here for further reference.

Reply:
That is a good idea!
Let's define the constellation of DUT and we do it in another ticket.